docs/docsite/rst/user_guide/complex_data_manipulation.rst

```diff
@@ -0,0 +1,243 @@
+.. _complex_data_manipulation:
+
+Complex data manipulation
```

Comment:
"Advanced" instead of "Complex" ? for me this is complex (because I'm not used to it) and advanced (because I never use that)

Reply:
I think data is complex and syntax is advance.